conduct.md

```diff
@@ -32,8 +33,20 @@ As a member of the community, you agree to the following:
 
 - Keep unstructured critique to a minimum. We encourage sharing ideas and perspectives, so please ensure that your feedback is constructive and relevant. If you have solid ideas you want to experiment with, make a fork and see how it works.
 - Avoid aggressive and micro-aggressive behavior, such as unconstructive criticism, providing corrections that do not improve the conversation (sometimes referred to as "well actually"s), repeatedly interrupting or talking over someone else, feigning surprise at someone’s lack of knowledge or awareness about a topic, or subtle prejudice (for example, comments like “That’s so easy my grandmother could do it.”). For more examples of this kind of behavior, [see the Recurse Center's user manual](https://www.recurse.com/manual#sec-environment).
-- We will exclude you from interaction if you insult, demean or harass anyone. The term “Harassment” includes “Unacceptable Behavior” described in the [Citizen Code of Conduct](http://citizencodeofconduct.org/). **In particular, we don’t tolerate behavior that excludes people in socially marginalized groups.**
-- Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member's behavior, please [contact one of the moderators](#contact) immediately.
+- We will exclude you from interaction if you insult, demean or harass anyone. The term “Harassment” includes “Unacceptable Behavior” described in the [Citizen Code of Conduct](http://citizencodeofconduct.org/):
```

Comment:
Maybe completely drop what is or isn't included in the term Harassment. It's irrelevant and may invite rule-lawyering.
```suggestion
- We will exclude you from interaction if you insult, demean or harass anyone. Behaviour that will lead to exclusion includes the following:
```

Reply:
this seems like a sensible adjustment, clarifying what I understand to be the intended meaning anyway. and it's within the scope of this PR. @TimMoore wdyt?

Reply:
Yes, I agree. I'll allow another couple of days for further comments from the community before committing this suggestion.

Reply:
You get rules-lawyering whether or not you specify the rules.  If adding rules adds clarity, go for it.  If it is misleading, drop it.  If you're worried about rules-lawyering the only solution is to not have rules.

Reply:
@Ichoran just to make sure I understand: are you objecting to @martijnhoekstra's suggested change?

Reply:
I haven't formed an opinion on the change itself.  I am objecting to the stated reasoning.

In one place, we list age and gender and so on without fear that it will cause "rules-lawyering" or that it is irrelevant given that we welcome "all".

Here, the proposal is to drop even a link to a list to avoid "rules-lawyering".

This seems inconsistent.  So if there is a reason to remove lists, the stated reason doesn't quite seem to be it.

Reply:
I agree that "invites rule-lawyering" isn't the best defense of Martijn's tweak. I simply think that Martijn's tweak makes the wording clearer.

As for the main change here, Rex I think you may have misunderstood the nature of the change, probably because you are looking at just this one point in the diffs. If you look at the whole diff, you'll see that "the proposal is to drop even a link to a list" simply isn't what's happening here. Rather, the link is being replaced by an inlined list.

Reply:
Ah, okay.  'You're right; I'd thought the diff would show the following stuff.  Sounds reasonable to inline the list, and being clearer is enough!

However, are we giving appropriate credit if we remove the link?

Reply:
> are we giving appropriate credit if we remove the link?

see credits at the bottom. our code already incorporates big chunks of text from the other codes, and Citizen CoC is already listed.

Reply:
Thanks for clarifying this, @SethTisue. I agree that with the list inlined, the link is no longer so important, and the suggested change adds clarity. With Citizen Code of Conduct explicitly acknowledged in the credits, I believe we've done our duty of attribution.

@Ichoran, please let me know if you're satisfied with this explanation, and if so I'll accept @martijnhoekstra's change to the lead sentence, assuming there are no other objections.

As a separate issue, I noticed that the Citizen Code of Conduct is licensed under the [CC Attribution-ShareAlike license](https://creativecommons.org/licenses/by-sa/3.0/). This is a GPL-like license that requires that derived works be distributed under the same license. In my interpretation, this should have been true all along, since the Scala CoC built upon the Citizen CoC in a way, even before inlining the content. Currently, there's no explicit license documented, but the Domain section implies permission to reuse the Scala CoC, and of course in practice other projects have adopted it. So, I think the spirit of that license is embraced, but abiding strictly by its terms would require explicitly documenting the use of that license, and also requiring other users of the Scala CoC to use the same license.

Reply:
I'll raise another pull request to add that license, and we can discuss that specific issue there.

Reply:
https://github.com/scala/scala-lang/pull/1082 I think that will need to be resolved before this one can be merged. Should I also start a thread on Discourse?

Reply:
I don't think a Discourse thread is necessary, I don't think there is broad interest in what the license on the Scala CoC itself is or might be

Reply:
I'll go ahead and commit the suggestion from @martijnhoekstra now.

Reply:
I merged the PR with the new license.